### PR TITLE
Test add

### PR DESCRIPTION
**Pull request by diamdesign**
Testing to add a comment in the css file to update and pull merge git.

### CODE REVIEW

**Comment by Cloticc** [comment]
Helt ok

**Comment by mansooriabbas** [comment]
Looks good !!

**Review by Cloticc — APPROVED** [review]
Nice

**Review by AlirezaKToosi — COMMENTED** [review]
bra jobbat!

**Comment by ksarvanitakis** [comment]
Bra bra!

File: createmap/src/App.css [hunk 1]
```diff
@@ -299,3 +299,5 @@ body,
 .gridless .grid-row {
 	border: none;
```
Comment (mansooriabbas) [inline createmap/src/App.css:300]: whats this?

File: createmap/src/App.css [hunk 2]
```diff
@@ -299,3 +299,5 @@ body,
...
 }
+
+/* Hello there */
```
Comment (diamdesign) [inline createmap/src/App.css:303]: Really cool comment!

**Review by mansooriabbas — CHANGES_REQUESTED** [review]

File: createmap/src/App.css (hunk 1 above)
Comment (diamdesign) [reply createmap/src/App.css:300]: It is a class to remove the grid borders in the create map grid layout.

**Pull request merged**